tests/eventGenerators.ts

```diff
@@ -35,19 +52,52 @@ export function generateTransferSingle(operator: string, from: string, to: strin
     mockEvent.logType,
     mockEvent.block,
     mockEvent.transaction,
-    mockEvent.parameters
+    mockEvent.parameters,
+    mockEvent.receipt
```

Comment:
🎨 All of these `mockEvent.receipt` were necessary additions to the `graph-ts`'s `ethereum.Event` object. The `.receipt` attribute was added to the latest `matchstick-as` version.